conans/test/functional/generators/virtualbuildenv_test.py

```diff
@@ -21,11 +21,21 @@ class TestConan(ConanFile):
         client.save({"conanfile.py": conanfile})
         client.run('install . -g virtualbuildenv -s os=Windows -s compiler="Visual Studio"'
                    ' -s compiler.runtime=MD -s compiler.version=15')
-        bat = load(os.path.join(client.current_folder, "activate_build.bat"))
-        self.assertIn("SET UseEnv=True", bat)
-        self.assertIn('SET CL=-MD -DNDEBUG -O2 -Ob2 %CL%', bat)
+        bat = load(os.path.join(client.current_folder, "environment_build.bat.env"))
+        self.assertIn("UseEnv=True", bat)
+        self.assertIn('CL=-MD -DNDEBUG -O2 -Ob2 %CL%', bat)
 
     def environment_deactivate_test(self):
+        if platform.system() == "Windows":
+            """ This test fails. The deactivation script takes the value of some envvars set by
+                the activation script to recover the previous values (set PATH=OLD_PATH). As this
+                test is running each command in a different shell, the envvar OLD_PATH that has
+                been set by the 'activate' script doesn't exist when we run 'deactivate' in a
+                different shell...
+
+                TODO: Remove this test
```

Comment:
Vote for removal

Reply:
Sounds like a useless test, but to confirm, shouldn't linux and mac be affected also?

Reply:
IMO this test is useless as it is already tested. In this test we are already checking that  the full environment is equal before activating and after deactivating the environment:

https://github.com/conan-io/conan/blob/1825f808b37724b37cda9f0114840d0d5b3d55c8/conans/test/functional/generators/virtualenv_test.py#L191